source/mapping-rules/mapping-rule-options/time-series.txt

```diff
@@ -0,0 +1,122 @@
+.. _rm-timeseries:
+
+===========
+Time Series
```

Comment:
[Question]

Since this is a task page, should this page title begin with a verb? 

Per the [style guide](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/task/): "Task page titles begin with a present-tense action verb."